packages/gamut/src/Form/Form.tsx

```diff
@@ -1,14 +1,33 @@
+import { system } from '@codecademy/gamut-styles';
+import { StyleProps, variance } from '@codecademy/variance';
+import styled from '@emotion/styled';
 import cx from 'classnames';
 import React, { FormHTMLAttributes } from 'react';
 
 import styles from './styles/Form.module.scss';
 
-export type FormProps = FormHTMLAttributes<HTMLFormElement> & {
-  className?: string;
-};
+const formSystemProps = variance.compose(
+  system.space,
+  system.border,
+  system.layout,
+  system.color,
+  system.shadow,
+  system.positioning,
+  system.background,
+  system.typography,
+  system.flex,
+  system.grid
+);
+
+export type FormProps = FormHTMLAttributes<HTMLFormElement> &
+  StyleProps<typeof formSystemProps> & {
+    className?: string;
+  };
+
+const StyledForm = styled('form')<FormProps>(formSystemProps);
```

Comment:
Added back the type guard and filter here - also theres some inference we can use that would be nice :)

Also I'm not sure we need all of these props - so I pruned it down to the abstract layout ones - feel free to ignore if you think we'll need the others presently :)

```suggestion
const formSystemProps = variance.compose(
  system.space,
  system.border,
  system.layout,
  system.positioning,
  system.flex,
  system.grid
);

export type FormProps = ComponentProps<typeof StyledForm>

const StyledForm = styled('form', styledOptions<'form'>())(formSystemProps);
```